moto/resourcegroups/models.py

```diff
@@ -350,6 +359,16 @@ def update_group_query(self, group_name, resource_query):
         self.groups.by_name[group_name].resource_query = resource_query
         return self.groups.by_name[group_name]
 
+    def get_group_configuration(self, group_name):
+        group = self.groups.by_name.get(group_name)
+        configuration = group.configuration
+        return configuration
+
+    def put_group_configuration(self, group_name, configuration):
+        if configuration:
```

Comment:
Is this a valid scenario? I'm assuming that the Configuration-param is required when calling this method

Reply:
Done